recipe/disable-backtrace-with-lines.patch

```diff
@@ -12,8 +12,8 @@ index 578f5c0..26f6355 100644
  #else
      char **syms = backtrace_symbols(trace, n);
      if (syms) {
- 	int i;
+         int i;
 +        fprintf(stderr, "conda-forge::ruby can't rb_dump_backtrace_with_lines\n");
```

Comment:
there's some reason why we can't print the backtraces? if yes, keeping conda-forge in the message looks weird.

Reply:
This patch was added for passing `make check `(for v2.6.3) https://github.com/conda-forge/ruby-feedstock/pull/24/commits/d6479ef20908bd8eaf46c2973f01ffb400929b6e#diff-10867a4e7bf6316ac62bc719d8aadcf92b47720d5a97aa5d00b3573b4ee96088R16 and applied for our recipe (v3.1.2) https://github.com/AnacondaRecipes/ruby-feedstock/commit/ff7ea780b19ddca713c4916a45f63b8a5dce3014.

I can remove the mention of conda-forge from the patch.

Reply:
I found only this very old bug report: https://bugs.ruby-lang.org/issues/9654

does the error still happen? can we fix the error instead of dropping functionality?

Reply:
I think the Bug #9654 was fixed https://docs.ruby-lang.org/en/2.4.0/ChangeLog-2_2_0.html#label-Mon+Mar+24+13-3A13-3A36+2014++NARUSE-2C+Yui++-3Cnaruse-40ruby-lang.org-3E. And it was found on linux-ppc64le that we don't support anymore:
`ruby 2.2.0dev (2014-03-19) [powerpc64-linux]`
```
This error began to occur after this change:
http://www.rubyist.net/~kanemoto/chkbuild/plinux/ruby-trunk/log/20140314T070002Z.diff.html.gz
Due to this error, TestBugReporter#test_bug_reporter_add fails on ppc64 GNU/Linux.
```

Reply:
I removed it